tests/python/dashboard/test_python_import.py

```diff
@@ -24,6 +24,10 @@ def test_python_import(dashboard):
     """
     dashboard.load_example("testdata/example.py", manual=True)
 
+    # Wait for the dashboard to finish loading the example file
+    # This ensures all UI elements are rendered before we check their values
+    dashboard.sb.wait_for_element_present("#Input_route", timeout=10)
+

```

Comment:
Maybe even before starting (every) test...?
```suggestion
    # Wait for the dashboard to finish loading the example file
    # This ensures all UI elements are rendered before we check their values
    dashboard.sb.wait_for_element_present("#Input_route", timeout=10)

    dashboard.load_example("testdata/example.py", manual=True)

```

Reply:
Moved into `wait_for_interaction_ready`, which is used by the `dashboard` fixture we use